### PR TITLE
Add proper FHIR registered marks

### PR DESCRIPTION
**Pull request by daliboz**
The first, prominent mention on each page now uses the HL7 FHIR standard and the registered character, most mentions use FHIR standard/clarify that we're an implementation of the standard.

Removed the prominent FHIR logo.

### CODE REVIEW

File: content/dstu2.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ The `:ehr_source_id` represents the tenant for which data should be retrieved.
 
 ### Resource
 
-`:resource` represent the FHIR resource to access. Example: <a href="/dstu2/patient/">`Patient`</a>
+`:resource` represent the FHIR<sup>®</sup> standard resource to access. Example: <a href="/dstu2/patient/">`Patient`</a>
```
Comment (MaxPhilips) [inline content/dstu2.md]: Pluralize represent

File: content/faq.md [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
+The SMART (Substitutable Medical Apps & Reusable Technology) platform defines a specification for an EHR to safely and securely open other applications with context. These SMART applications are commonly web applications but may also be native mobile applications and will use the FHIR<sup>®</sup> standard to read and write data from the EHR. With SMART, Cerner can embed a SMART app in the EHR. Cerner believes that SMART applications will be a major user of implementations of the FHIR<sup>®</sup> standard resources. We will also support access through mobile SMART applications, as those specifications emerge from the [SMART](http://smartplatforms.org/) web site.
 
 ### Where can I ask a question?
 
-A developer can post a question to the community using the [Cerner FHIR Developers](https://groups.google.com/forum/#!forum/cerner-fhir-developers) group.
+A developer can post a question to the community using the [our google group](https://groups.google.com/forum/#!forum/cerner-fhir-developers).
```
Comment (MaxPhilips) [inline content/faq.md]: Delete the

**Comment by MaxPhilips** [comment]
All the lines with ® in super script take up a lot of extra space on top of the text.  This looks weird but I'm not sure if there's a way around it.  I tried shrinking the font size on the symbol in superscript but it still took the same amount of vertical space.

**Comment by daliboz** [comment]
It looks like I may be able to play with css to fix the line height issue. I'll try a few things...

File: content/dstu2.md (hunk 1 above)
Comment (daliboz) [inline content/dstu2.md]: fixed

File: content/faq.md (hunk 2 above)
Comment (daliboz) [inline content/faq.md]: fixed

**Comment by MaxPhilips** [comment]
+1

**Comment by daliboz** [comment]
Quick review on the error correction commit as well - noticed that we were getting errors on loading the current page because of some bad styles and missing files.

**Comment by MaxPhilips** [comment]
+1, no more console errors loading the site locally

**Comment by davischristi** [comment]
Verified running locally.
👍

**Pull request merged**